modules/manage/pages/manage-security/configure-server-certificates.adoc

```diff
@@ -241,175 +147,229 @@ keyUsage = digitalSignature,keyEncipherment
+Remember that you must copy the node's certificate and key files to its own `inbox` directory to deploy them.
 
 [#root-intermediate-and-node-certificates]
-== Cluster Protection with Root, Intermediate, and Node Certificates
+== Create and Use Intermediate Certificates to Sign Node Certificates 
 
-Optionally, a root certificate can be used to sign an _intermediate_ certificate, which is then itself used to sign node certificates.
-This increases security, since it minimizes use of the private key associated with the root certificate, when many node or client certificates are to be signed.
+The previous example directly signed node certificates using the cluster certificate.
+In some cases, you may want to use an intermediate certificate to sign the certificates for the nodes.
+The primary reason to use an intermediate certificate is to prevent exposing the cluster's private key.
 
-When a client attempts to access the cluster securely, it must be able to build the certificate chain used to convey the authority of the cluster's root CA (see xref:learn:security/certificates.adoc#intermediate-certificates[Intermediate Certificates]).
-In Couchbase Server Version 7.1+, the certificates can be prepared for client inspection in either of two ways, which are:
+For example, you may want to delegate the signing of node certificates.
+By creating an intermediate certificate, you can keep the cluster's private key secret while allowing others to sign node certificates.
+The administrators to whom you delegate the signing of node certificates can use the intermediate certificate for signing.
+They do not need use to the cluster's private key to sign the node certificates. 
 
-* Concatenation of all intermediate and node certificates into a single _chain.pem_ file, which is uploaded from the node's _inbox_.
+For more information, see xref:learn:security/using-multiple-cas.adoc#adding-intermediate-certificates-to-the-trust-store[Adding Intermediate Certificates to the Trust Store].
 
-* Upload, from the node's inbox, of a _chain.pem_ file containing only the unconcatenated node certificate; with the expectation that each intermediate certificate resides in the client's _trust store_.
+When a client attempts to connect to the database securely, it uses the node's certificate to verify the node's identity.
+For it to verify the node identity, the client searches for a CA it trusts in the chain of certificates from the node, through any intermediates, to the cluster certificate.
+See xref:learn:security/certificates.adoc#intermediate-certificates[Intermediate Certificates] for more information.
 
-Both procedures are shown below.
-For more information, see xref:learn:security/using-multiple-cas.adoc#adding-intermediate-certificates-to-the-trust-store[Adding Intermediate Certificates to the Trust Store].
+In Couchbase Server Version 7.1 and later, you can supply the client with the chain of trust it requires in one of two ways:
```

Comment:
Do clients not use the node's certificate to determine the identity of the node? If so, I would think that this duscussion applies to both other nodes and clients. What if I use the term "peer" to indicat ethis can be another node, a client, etc?

Reply:
> Do clients not use the node's certificate to determine the identity of the node?

In general all components can use certificates for identity verification. Here we are talking about node configuration, right?
Also client is a very generic word and it can be very confusing. Not clear what is meant in this case. Is that SDK instance? or is that TLS client?

Since we are talking about node certificates here, and chain and trusted certificates are uploaded to nodes I think it is less confusing to just say " you can supply the node with the chain of trust with it requires in one of the following ways:"

Reply:
Actually you changed my mind about that, we probably can use peer here. Because in some cases it is used on the client side, sometimes on the server side. In some cases it can be SDKs, in some cases it is server node. Yes, I agree peer should be ok.